Chapter 9 (Python dictionary)/task_2.py

```diff
@@ -0,0 +1,10 @@
+#Task 2: Dictionary Iteration
+grades = {
+    "Alice": 85,
+    "Bob": 92,
+    "Charlie": 88,
+    "Diana": 95,
+    "Eve": 78
+}
+for users in grades:
+    print(f" {users}:{grades[users]}")
```

Comment:
```suggestion
for name, grade in grades.items():
    print(f" {name}:{grade}")
```
This way we avoid getting value from `grades` each time